i18n/fa.yaml

```diff
@@ -1,19 +1,24 @@
 - id: prev_page
-  translation: "صفحه قبلی"
+  translation: "صفحه قبل"
 
 - id: next_page
-  translation: "صفحه بعدی"
+  translation: "صفحه بعد"
 
 - id: read_time
   translation:
-    one: "۱ دقیقه"
+    one : "1 دقیقه"
     other: "{{ .Count }} دقیقه"
 
+- id: words
+  translation:
+    one : "کلمه"
+    other: "{{ .Count }} کلمه"
+
 - id: toc
   translation: "فهرست مطالب"
 
 - id: translations
-  translation: "ترجمه ها"
+  translation: "ترجمه‌ها"
```

Comment:
I think the correct form would be using half-space. Ref: https://utype.ir/blog/%D9%86%DB%8C%D9%85-%D9%81%D8%A7%D8%B5%D9%84%D9%87-%D9%81%D8%A7%D8%B1%D8%B3%DB%8C/#:~:text=%DB%B5%2D%20%D8%AF%D8%B1%20%D8%B2%D9%85%D8%A7%D9%86,%DA%A9%D8%AA%D8%A7%D8%A8%E2%80%8C%D9%87%D8%A7%D8%8C%20%D8%AE%D8%A7%D9%86%D9%87%E2%80%8C%D9%87%D8%A7

Reply:
I don't know if you've changed the code or if I was confused before. Anyway, you're doing the right thing now.